src/App.jsx

```diff
@@ -14,45 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goods, setGoods] = useState(initialGoods);
+  const [sortAlphabeticallyLight, setSortAlphabeticallyLight] = useState(true);
+  const [sortByLengthLight, setSortByLengthLight] = useState(true);
+  const [reverseLight, setReverseLight] = useState(true);
+  const [resetLight, setResetLight] = useState(true);
+  const [ascending, setAscending] = useState(true);
+
+  const sortAlphabetically = () => {
+    const sortedGoods = [...goods].sort();
```

Comment:
You are sorting goods directly which can lead to unstable sorting when items are equal. Consider using a stable sorting algorithm or adding additional conditions.